### ISSUE TITLE
Essential addons not working

### ISSUE THREAD

**Issue by vojinmaletkovic** [WPDevelopers/essential-addons-for-elementor-lite#669]
Essential addons plugin is not working with elementor. I tried to activate on two my wordpress sites. What is happening?

**Comment by abid112** [comment]
Hi @vojinmaletkovic ,

Can you please tell me what exact issue you're facing actually?
Please share a screenshot so that I can understand better

Thank you!

**Comment by vojinmaletkovic** [comment]
Hello,

My elementor can't load. I am sending you pictures. It's all the same on all three Web adress. When I deactivate Essential Addons plugin it works Just fine.

Best regards,

Vojin Maletkovic
![IMG_20210712_135856](https://user-images.githubusercontent.com/87196115/125284499-cbee1f00-e319-11eb-847c-9e03707dd9e7.jpg)

**Comment by abid112** [comment]
Hi @vojinmaletkovic 

It's possible that your website is having low memory. You can check your memory limit by navigating to wp-admin -> Elementor -> System Info. Check out this screenshot: https://d.pr/WVKfum

You can install the following plugin on your website to take care of this: https://d.pr/f/D3Dik2

If that doesn't work, make sure to increase the memory limit. To increase the memory limit you will have to modify the wp-config.php file yours. Just above this line " /* That's all, stop editing! Happy blogging. */" add this following line of code:

define('WP_MEMORY_LIMIT', '512M');

You would find the wp-config.php file under your WordPress directory.

You can also read this blog to modify the wp-config.php file: https://wpdeveloper.net/troubleshoot-preview-loading-error-elementor/

If that doesn't work, please deactivate the Essential Addons elements you are not using especially the Dynamic Elements.

Let me know how it goes.
Thank you!

**Comment by vojinmaletkovic** [comment]
Thank you for your comment, but that is not solution of this problem because my memory limit is 256M and domains don't have problem with that.

**Comment by abid112** [comment]
Hi @vojinmaletkovic , 

256M is actually a low memory if you've enabled Dynamic Elements. Also if you use other plugins, that would allocate the memory aswell on your website.
So I would recommend you increase the memory limit to 512M atleast to use Essential Addons for Elementor.

Thank you!

**Comment by vojinmaletkovic** [comment]
Yes, I did that, but the problem is still the same...

**Comment by abid112** [comment]
Hi @vojinmaletkovic ,

Can you please create a support ticket with temporary admin access from here: https://wpdeveloper.net/support/new-ticket/

We will deeply check this issue on your website and get back to you there don't worry.

Thank you!

**Comment by vojinmaletkovic** [comment]
Of course, but I am currently on vacation and when I get back I Will do it.

Thank you for your support!

Best regards.

**Comment by abid112** [comment]
You're welcome!

**Comment by vojinmaletkovic** [comment]
> You're welcome!

Here you go the log in to my wordpress site. This is my domain for practice, but it's the same problem on every domain that I have. Wordpress is on english here.
bistroweb.in.rs/wp-admin
user: bokivoki
pass: bokivoki2021

**Comment by abid112** [comment]
Hi @vojinmaletkovic ,

Hope you're doing good.
First of all, I would recommend you to not provide admin access in GitHub. This is a public/open forum. We have a separate support channel for this: https://wpdeveloper.net/support/new-ticket/
We've deleted your previous comment(admin access)

Well, I've investigated your website and found that **WooCommerce Cart Abandonment Recovery** and WooLentor - **WooCommerce Elementor Addons + Builder** in conflicting actually! I've deactivated these two plugins and it seems now the Elementor panel perfectly loads on your edit page.

Also, I've noticed that you're using the outdated version of these two plugins. So I would recommend you to update them aswell. Make sure to backup your website before the update.

Thank you!
